lib/train/transports/vmware.rb

```diff
@@ -34,7 +34,7 @@ def initialize(options)
         @powershell_binary = detect_powershell_binary
 
         if @powershell_binary == :powershell
-          require_relative "local"
```

Comment:
This should stay as require_relative which is higher performance

require_relative "transports/local"

Reply:
@tas50 require_relative is not working here as it's unable to load the local.rb file from the train-core gem.